cmd/registry/patch/version.go

```diff
@@ -66,15 +66,39 @@ func newApiVersion(ctx context.Context, client *gapic.RegistryClient, message *r
 	}, nil
 }
 
+func applyApiVersionPatchBytes(
+	ctx context.Context,
+	client connection.RegistryClient,
+	bytes []byte,
+	parent string) error {
```

Comment:
This function can easily be defined in one line. We have much longer function definitions throughout the codebase, and defining it on one line prevents indentation confusion. This style causes the function definition to blend in with the code it contains. More details messaged privately.

Reply:
Thanks for the link. As written this is consistent with at least one other instance in this file and several more in parallel. I think those should be addressed together (and separately). I filed #803 and there counted 46 instances of multiline declarations.